Doc/library/zipfile.rst

```diff
@@ -351,13 +365,15 @@ ZipFile Objects
       :exc:`ValueError`.  Previously, a :exc:`RuntimeError` was raised.
 
 
-.. method:: ZipFile.write(filename, arcname=None, compress_type=None)
+.. method:: ZipFile.write(filename, arcname=None, compress_type=None, \
+                          compress_level=None)
 
    Write the file named *filename* to the archive, giving it the archive name
    *arcname* (by default, this will be the same as *filename*, but without a drive
    letter and with leading path separators removed).  If given, *compress_type*
    overrides the value given for the *compression* parameter to the constructor for
-   the new entry.
+   the new entry. Similarly, if given, *compress_level* overrides the *compresslevel*
+   parameter.
```

Comment:
Spelling is too similar yet inconsistent. Wouldn’t it be better to remove the underscore from all parameters, and differentiate them as method parameters versus the constructor parameter?

Reply:
I was following the example of `compression` (constructor's argument) to `compress_type` (method's argument). It's not clear to me why those are different, but if it's OK for `compresslevel` to be the same throughout that's fine by me?